server/src/com/cloud/alert/AlertManagerImpl.java

```diff
@@ -767,7 +767,9 @@ public void sendAlert(AlertType alertType, long dataCenterId, Long podId, Long c
                 // set up a new alert
                 AlertVO newAlert = new AlertVO();
                 newAlert.setType(alertType.getType());
-                newAlert.setSubject(subject);
+                //do not have a seperate column for content.
+                //appending the message to the subject for now.
+                newAlert.setSubject(subject+content);
```

Comment:
@rafaelweingartner 
I agree with you but "what can be done today" depends on the individuals priorities and bandwidth. Just so that the required improvement dose not get lost in comments i have created a ticket in Jira to track this, CLOUDSTACK-9846.